.circleci/config.yml

```diff
@@ -109,8 +109,8 @@ workflows:
               only: tx-pull-manual
   weekly-tx-pull:
       triggers:
-        - schedule: # weekly on Tuesdays at 8pm
-            cron: "0 20 * * 2"
+        - schedule: # weekly on Wednesday at 3am
+            cron: "0 03 * * 3"
```

Comment:
I don't know if it's actually a problem, but you might want to use `3` instead of `03`. It sounds like some implementations of cron don't like the leading zero: https://unix.stackexchange.com/a/579630/106290